.github/workflows/actions.yml

```
@@ -0,0 +1,57 @@
+      - uses: actions/setup-node@v4
+
+      - uses: docker://alpine:latest
+
+      - name: Cache untrusted dependency path
+        uses: actions/cache@v4
+        with:
+          path: node_modules
+          key: ${{ github.event.pull_request.title }}
+
+      - name: Template injection via PR title
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)